CHANGELOG.md

```diff
@@ -6,18 +6,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- [2321](https://github.com/FuelLabs/fuel-core/pull/2321): New metrics for the txpool:
```

Comment:
should this mention both #2321 and #2385?

Reply:
I think it should. I wanted to keep the "one PR per line" rule, so this is how I have changed it:
https://github.com/FuelLabs/fuel-core/pull/2385/commits/dadaaa6214160b9a331c55ac8bb13bc366a44433